project.clj

```diff
@@ -13,7 +13,8 @@
                  [secretary "1.2.3"]]
 
   :plugins [[lein-environ "1.0.2"]
-            [lein-asset-minifier "0.2.8"]]
+            [lein-asset-minifier "0.2.8"]
+            [lein-cljfmt "0.3.0"]]
```

Comment:
Just for the record, in case anyone else is reading this: appears lein-ancient doesn't check plugins. It had no complaints about the version of cljfmt.